### PR TITLE
docs: edits to guide section

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
This PR makes some major line edits to the "guides" section of the docs. In a few cases, I've added intros, or rewritten them to create a better flow between paragraphs and pages.

Some common edits are:
* Using bold for UI elements, italics for special terms, and monospace for variables and code
* rewriting passive tense into active tense
* breaking paragraphs up into distinct ideas, and using lists for instructions (following guidelines here: https://docs.microsoft.com/en-us/style-guide/scannable-content/)

### CODE REVIEW

File: docs/docs/guides/solution-scenarios/introduction.mdx [hunk 1]
```diff
@@ -11,8 +11,11 @@ import Column from "../../../src/components/column";
 
 ## Solution Scenarios
 
-Customers of an SaaS Identity and Access Management System usually have all distinct use cases and requirements.
-This guide attempts to explain real-world implementations and break them down into **Solution Scenarios** which aim to help you getting started with ZITADEL.
+Customers all have different use cases for their SaaS Identity and Access Management Systems.
+
+This guide outlines the real-world concepts you need to think about when implementing an Access Management System.
```
Comment (eliobischof) [inline docs/docs/guides/solution-scenarios/introduction.mdx:16]:
```suggestion
This guide outlines the real-world concepts you need to think about when integrating an Access Management System.
```

File: docs/docs/guides/trainings/supportservice/recurring.md [hunk 2]
```diff
@@ -4,14 +4,16 @@ title: Recurring Trainings
 
 ## ZITADEL Support Refresher
 
-In this session you can refresh knowledge about existing and gain experience with new features of ZITADEL to keep the quality of your support high. We recommend an one day training per support staff.
+In this session you can refresh your knowledge about existing concepts, and gain experience with new features of ZITADEL.
+This keeps the quality of your support high.
+We recommend an one day of training per support staff.
```
Comment (eliobischof) [inline docs/docs/guides/trainings/supportservice/recurring.md:9]:
```suggestion
We recommend one day of training per support staff.
```

**Review by eliobischof — CHANGES_REQUESTED** [review]

File: docs/docs/guides/basics/organizations.mdx [hunk 3]
```diff
@@ -16,51 +16,69 @@ import Column from '../../../src/components/column';
...
 
-An organization can have multiple domain names, but only one domain can be primary. The primary domain defines which login name ZITADEL displays to the user, and what information gets asserted in access_tokens (`preferred_username`).
+An organization can have multiple domain names, but only one domain can be primary.
+The primary domain defines which loginname ZITADEL displays to the user, and what information gets asserted in `access_tokens` (`preferred_username`).
 
-Please note that domain verification also removes the logonname from all users, who might have used this combination in the global organization (ie. users not belonging to a specific organization). Relating to our example with acme.ch: If a user ‘coyote’ exists in the global organization with the logonname coyote@acme.ch, then after verification of acme.ch, this logonname will be replaced with `coyote@{randomvalue.tld}`. ZITADEL will notify users affected by this change.
+Note that domain verification also removes the loginname from all users, some of whom might have used this combination in the global organization
```
[the remark this answers is not in the document]
Comment (fgerschwiler) [reply docs/docs/guides/basics/organizations.mdx:63]: yes

File: docs/docs/guides/authorization/oauth-recommended-flows.md [hunk 4]
```diff
@@ -29,52 +29,70 @@ title: Recommended authorization flows
...
 
-So this module will only go over the basics and explain why we recommend the flow “Authorization Flow with PKCE” as default for most applications. We will also cover the case of machine-to-machine communication, ie. where there is no interactive login. Further we will guide you to further reading viable alternatives, if the default flow is not feasible.
+We invite you to explore the different authorization flows in the OAuth 2.x standard.
+At the start, we assume that you have a brand-new application (ie. without legacy requirements),
+and that you found a reliable SDK/Library that does the heavy lifting for you.
+
+This module covers only the basics.
+It explains why we recommend the flow “Authorization Flow with PKCE” as default for most applications.
+We also cover the case of machine-to-machine communication, ie. where there is no interactive login.
+For cases where the default flow is not feasible.
```
[the remark this answers is not in the document]
Comment (fgerschwiler) [reply docs/docs/guides/authorization/oauth-recommended-flows.md:47]: at the moment we have only documented the recommended flows, some alternatives are not documented so the module will not redirect

[the remark this answers is not in the document]
Comment (eliobischof) [reply docs/docs/guides/authorization/oauth-recommended-flows.md:47]:
```suggestion
```

File: docs/docs/guides/solution-scenarios/b2b.mdx [hunk 5]
```diff
@@ -4,82 +4,116 @@ title: B2B
...
+Employees work for a department within Octagon, or for Octagon itself. 
+Some users supervise teams.
+These users have enhanced permissions, letting them onboard new employees and manage their roles and access.
 Target groups of the application can be split into:
-- **Employees:** users who are using the application as a starting point for their work.
-- **Supervisors:** users who are mainly using the application to manage users and their access of their department.
-- **Administrators:** this users are able to grant additional organizations or departments and elect supervisors.
+- Employees: users who use the application as a starting point for their work.
+- Supervisors: users who use the application to manage users and their access of their department.
+- Administrators: users who can grant additional organizations or departments and elect supervisors.
```
[the remark this answers is not in the document]
Comment (eliobischof) [reply docs/docs/guides/solution-scenarios/b2b.mdx:39]: > yes, but it sounds a little bit complicated to me

LOL, to me too :-)

[the remark this answers is not in the document]
Comment (eliobischof) [reply docs/docs/guides/solution-scenarios/b2b.mdx:39]: @juergrinaldi do you have an idea about how to simplify and/or illustrate the description for these groups and their capabilities?

**Review by eliobischof — APPROVED** [review]

**Pull request merged**